__tests__/lib/components/PrimaryButton-test.js

```diff
@@ -0,0 +1,51 @@
+jest.unmock('../../../lib/components/PrimaryButton.jsx');
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import {shallow} from 'enzyme';
+
+import PrimaryButton from '../../../lib/components/PrimaryButton.jsx';
+
+describe('PrimaryButton', () => {
+
+  let buttonItem = null;
+
+  const text = 'Test Label';
+
+  beforeEach(function() {
+    buttonItem = shallow(<PrimaryButton text={text} />);
+  });
+
+  it('is defined', () => {
+    expect(buttonItem).toBeDefined();
+  });
+
+  it('is a button element', () => {
+    expect(buttonItem.type()).toEqual('button');
+  });
+
+  it('is has specified text', () => {
+    expect(buttonItem.text()).toEqual(text);
+  });
+
+  it('is has a active class', () => {
+    buttonItem = shallow(<PrimaryButton text={text} status={PrimaryButton.STATUS_ACTIVE} />);
+    expect(buttonItem.hasClass('usa-button-active'));
+  });
+
+  it('is has a hover class', () => {
+    buttonItem = shallow(<PrimaryButton text={text} status={PrimaryButton.STATUS_HOVER} />);
```

Comment:
Is there a way using enzyme or jest to mock a mouse-over state rather than setting a prop this way?  How would this work on the actual page?  I doubt we want to add mouse-over and mouse-leave events on the button to re-render with these props for active and hover...